src/features/app/store.js

```diff
@@ -31,9 +31,15 @@ const store = configureStore({
 // settings across page loads; don't forget to toggle false when done testing!
 const persistState = false
 
+// if you want to save a multiple temporary states, use these keys. The first time
+// you save a new state, change persistSaveKey. Make a change, then change
+// persistInitKey to the same value. It's like doing a "save as"
+const persistInitKey = 'state'
+const persistSaveKey = 'state'
```

Comment:
Can you change these things in the developer console?

Reply:
Not at the moment. This will largely go away once we support Sandify export format.